tasks/viderman_a_elem_vec_sum/mpi/src/ops_mpi.cpp

```diff
@@ -0,0 +1,90 @@
+#include "viderman_a_elem_vec_sum/mpi/include/ops_mpi.hpp"
+
+#include <mpi.h>
+
+#include <cstddef>
+#include <vector>
+
+#include "viderman_a_elem_vec_sum/common/include/common.hpp"
+
+namespace viderman_a_elem_vec_sum {
+
+VidermanAElemVecSumMPI::VidermanAElemVecSumMPI(const InType &in) {
+  SetTypeOfTask(GetStaticTypeOfTask());
+  GetInput() = in;
+  GetOutput() = 0.0;
+}
+
+bool VidermanAElemVecSumMPI::ValidationImpl() {
+  return (GetOutput() == 0.0);
+}
+
+bool VidermanAElemVecSumMPI::PreProcessingImpl() {
+  return true;
+}
+
+bool VidermanAElemVecSumMPI::RunImpl() {
```

Comment:
Please, ensure the data is split and properly scattered across different processes